paper/README.qmd

```diff
@@ -256,7 +256,7 @@ Candidate matches as determined by intersecting AABBs must then be further evalu
 
 ```{r}
 #| fig-cap: "Matched lines with 15° angle tolerance and 2.5 meter distance tolerance."
-mtx <- rnetmatch::rnet_match(rnet_x, rnet_y, 2.5, 15)
+mtx <- anime::rnet_match(rnet_x, rnet_y, 2.5, 15)
```

Comment:
This is still broken since these functions don't exist. But it's fine anyways!

Reply:
Yeah... Will make further fixes in future, it's a bit less wrong (correct pkg name at least).